.github/pull-request-template.md

```diff
@@ -0,0 +1,15 @@
+<!-- General PR guidelines:
+
+Most PRs should be opened against the master branch.
+
+ -->
+
+## Proposed Changes
+
+- 
```

Comment:
Want to just keep 1 bullet? If someone is saying 3 things should be changed in one issue, that issue is probably too big